__tests__/app.test.js

```diff
@@ -120,3 +120,45 @@ describe("GET /api/articles/:id", () => {
   });
 });
 
```

Comment:
One missing test, if we request /api/articles/100000/comments it should give you back a 404. You made a checkExists / Promise.all to differentiate between valid articles with no comments: rows =[] and invalid articles: rows = [] 

This was covered in yesterday's lecture (I think, I wasn't there)